core/src/main/java/org/hisp/dhis/android/core/program/ProgramStage.java

```diff
@@ -76,40 +76,40 @@
+    private static final Field<ProgramStage, Boolean> captureCoordinates = Field.create(CAPTURE_COORDINATES);
+    private static final Field<ProgramStage, FormType> formType = Field.create(FORM_TYPE);
+    private static final Field<ProgramStage, Boolean> displayGenerateEventBox = Field.create(DISPLAY_GENERATE_EVENT_BOX);
+    private static final Field<ProgramStage, Boolean> generatedByEnrollmentDate =
             Field.create(GENERATED_BY_ENROLMENT_DATE);
-    public static final Field<ProgramStage, Boolean> autoGenerateEvent = Field.create(AUTO_GENERATE_EVENT);
-    public static final Field<ProgramStage, Boolean> hideDueDate = Field.create(HIDE_DUE_DATE);
-    public static final Field<ProgramStage, Boolean> blockEntryForm = Field.create(BLOCK_ENTRY_FORM);
-    public static final Field<ProgramStage, Integer> minDaysFromStart = Field.create(MIN_DAYS_FROM_START);
-    public static final Field<ProgramStage, Integer> standardInterval = Field.create(STANDARD_INTERVAL);
-    public static final Field<ProgramStage, PeriodType> periodType = Field.create(PERIOD_TYPE);
-    public static final Field<ProgramStage, Boolean> remindCompleted = Field.create(REMIND_COMPLETED);
-
-    public static final NestedField<ProgramStage, ProgramStageSection> programStageSections
+    private static final Field<ProgramStage, Boolean> autoGenerateEvent = Field.create(AUTO_GENERATE_EVENT);
+    private static final Field<ProgramStage, Boolean> hideDueDate = Field.create(HIDE_DUE_DATE);
+    private static final Field<ProgramStage, Boolean> blockEntryForm = Field.create(BLOCK_ENTRY_FORM);
+    private static final Field<ProgramStage, Integer> minDaysFromStart = Field.create(MIN_DAYS_FROM_START);
+    private static final Field<ProgramStage, Integer> standardInterval = Field.create(STANDARD_INTERVAL);
+    private static final Field<ProgramStage, PeriodType> periodType = Field.create(PERIOD_TYPE);
+    private static final Field<ProgramStage, Boolean> remindCompleted = Field.create(REMIND_COMPLETED);
+
+    private static final NestedField<ProgramStage, ProgramStageSection> programStageSections
             = NestedField.create(PROGRAM_STAGE_SECTIONS);
-    public static final NestedField<ProgramStage, ProgramStageDataElement> programStageDataElements =
+    private static final NestedField<ProgramStage, ProgramStageDataElement> programStageDataElements =
             NestedField.create(PROGRAM_STAGE_DATA_ELEMENTS);
-    public static final NestedField<ProgramStage, ObjectStyle> style = NestedField.create(STYLE);
-    public static final NestedField<ProgramStage, ObjectWithUid> program = NestedField.create(PROGRAM);
-    public static final NestedField<ProgramStage, Access> access = NestedField.create(ACCESS);
+    private static final NestedField<ProgramStage, ObjectStyle> style = NestedField.create(STYLE);
+    private static final NestedField<ProgramStage, ObjectWithUid> program = NestedField.create(PROGRAM);
```

Comment:
I have used ObjectWithUid just for the NestedField but not for the actual property to prevent static initialization errors, since Program requires PogramStage and ProgramStage requires Program.